src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1725,18 +1767,13 @@ public async Task<JArray> StackFrameGetValues(SessionId sessionId, MethodInfo me
 
             if (await IsAsyncMethod(sessionId, method.DebuggerId, token))
             {
+                List<int> objectAlreadyRead = new List<int>();
```

Comment:
```suggestion
                List<int> objectAlreadyRead = new();
```

Reply:
nit: pass the objectId to the new method, and the list can be created, and used in that method.
nit: rename to plural - `objectsAlreadyRead`

Reply:
done.